js/ionic.md

```diff
@@ -200,6 +200,8 @@ The component will emit two events:
 
  - `(picked)` - Emitted when an image is selected. The event will contain the `File` object which can be used for upload.
  - `(loaded)` - Emitted when an image preview has been rendered and displayed.
+ - `path` - An optional S3 image path (prefix).
+ - `storageOptions` - An object which is passed as the 'options' parameter to the .put request.  This can be used to set the 'level' of the objects being uploaded (i.e. 'protected', 'private', or 'public').
```

Comment:
Update text:

An object passed within the ‘options’ property in the Storage.put request. This can be used to set the permissions ‘level’ property of the objects being uploaded i.e. ‘private’, ‘protected’, or ‘public’.

Reply:
Also add a learn more link to link to storage permissions info/docs here

Reply:
Done!